[chassis] test_ser failing on T2 testbeds

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/sonic-net/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Describe the results you received:**
The `test_ser` tests fails on the T2 testbed with below error
```
        if (res.is_failed or &apos;exception&apos; in res) and not module_ignore_errors:
&gt;           raise RunAnsibleModuleFail(&quot;run module {} failed&quot;.format(self.module_name), res)
E           RunAnsibleModuleFail: run module shell failed, Ansible Results =&gt;
E           {&quot;changed&quot;: true, &quot;cmd&quot;: &quot;python /tmp/ser_injector.py -n 1&quot;, &quot;delta&quot;: &quot;0:00:01.394462&quot;, &quot;end&quot;: &quot;2022-12-04 08:51:58.922667&quot;, &quot;failed&quot;: true, &quot;msg&quot;: &quot;non-zero return code&quot;, &quot;rc&quot;: 1, &quot;start&quot;: &quot;2022-12-04 08:51:57.528205&quot;, &quot;stderr&quot;: &quot;Traceback (most recent call last):\n  File \&quot;/tmp/ser_injector.py\&quot;, line 902, in &lt;module&gt;\n    main()\n  File \&quot;/tmp/ser_injector.py\&quot;, line 897, in main\n    rc = serTest.test_memory(args.completeness)\n  File \&quot;/tmp/ser_injector.py\&quot;, line 612, in test_memory\n    self.enable_mem_scan(MEMORY_SCAN_INTERVAL_USEC, MEMORY_SCAN_ENTRIES)\n  File \&quot;/tmp/ser_injector.py\&quot;, line 697, in enable_mem_scan\n    self.enable_memory_scan(&apos;memscan&apos;, interval_usec, rate)\n  File \&quot;/tmp/ser_injector.py\&quot;, line 689, in enable_memory_scan\n    raise ValueError(&apos;Failed to start memory scanner: %s&apos; % cmd)\nValueError: Failed to start memory scanner: memscan&quot;, &quot;stderr_lines&quot;: [&quot;Traceback (most recent call last):&quot;, &quot;  File \&quot;/tmp/ser_injector.py\&quot;, line 902, in &lt;module&gt;&quot;, &quot;    main()&quot;, &quot;  File \&quot;/tmp/ser_injector.py\&quot;, line 897, in main&quot;, &quot;    rc = serTest.test_memory(args.completeness)&quot;, &quot;  File \&quot;/tmp/ser_injector.py\&quot;, line 612, in test_memory&quot;, &quot;    self.enable_mem_scan(MEMORY_SCAN_INTERVAL_USEC, MEMORY_SCAN_ENTRIES)&quot;, &quot;  File \&quot;/tmp/ser_injector.py\&quot;, line 697, in enable_mem_scan&quot;, &quot;    self.enable_memory_scan(&apos;memscan&apos;, interval_usec, rate)&quot;, &quot;  File \&quot;/tmp/ser_injector.py\&quot;, line 689, in enable_memory_scan&quot;, &quot;    raise ValueError(&apos;Failed to start memory scanner: %s&apos; % cmd)&quot;, &quot;ValueError: Failed to start memory scanner: memscan&quot;], &quot;stdout&quot;: &quot;&quot;, &quot;stdout_lines&quot;: []}
```

**Describe the results you expected:**
 This test should be skipped of T2 testbed because the Asics don't this test

**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
+++ b/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
@@ -642,7 +642,8 @@ platform_tests/sfp/test_sfputil.py::test_check_sfputil_reset:
   skip:
     reason: "platform does not support sfp reset"
     conditions:
-      - "platform in ['x86_64-cel_e1031-r0']"
+      - "platform in ['x86_64-cel_e1031-r0'] or 't2' in topo_name"
+      - https://github.com/sonic-net/sonic-mgmt/issues/6218
 
 platform_tests/test_auto_negotiation.py:
   skip:

